static/js/notifications.js

```diff
@@ -753,11 +753,7 @@ exports.handle_global_notification_updates = function (notification_name, settin
         page_params[notification_name] = setting;
     }
 
-    if (
-        settings_notifications.all_notifications.settings.stream_notification_settings.includes(
-            notification_name
-        )
-    ) {
+    if (settings_notifications.stream_notification_settings.includes(notification_name)) {
```

Comment:
I think this change might be a bug; the `wildcard_mentions_notify` setting won't update now.

Reply:
It does update `wildcard_mentions_notify` too, because of this change in the same commit (I just tested),
```js
exports.stream_notification_settings = general_stream_notification_settings.concat(
    other_stream_notification_settings);
```
but now I see there is no need to split `stream_notification_settings`.